documentation/source/reference.rst

```diff
@@ -0,0 +1,189 @@
+Json library reference
```

Comment:
"json Library Reference"

Reply:
is ok "Json Library Reference"?

Reply:
I think "Json" is a very odd capitalization for an acronym, but it's up to you.

Reply:
Fixed